crates/arkworks/src/bigint_256.rs

```diff
@@ -103,12 +106,16 @@ pub fn caml_bigint_256_compare(x: WasmBigInteger256, y: WasmBigInteger256) -> i8
     }
 }
 
-#[wasm_bindgen]
+// I don't think this is used anywhere
```

Comment:
It is required in `pasta_bindings.ml` on the Mina side, see [here](https://github.com/MinaProtocol/mina/blob/compatible/src/lib/crypto/kimchi_bindings/stubs/pasta_bindings.ml#L18). When we compile Mina, we need to have this stub. The `print` statement might be used in the Caml code in different places.